### DISCUSSION TITLE
Is it possible to use an existing pdf as startingpoint / template and then add things to it?

### DISCUSSION THREAD

**Discussion by Taz-1999** [diegomura/react-pdf#2084] [Q&A]
For instance if I have some nice templates for invoices, or letterheads and such. And want to programmatically add some things to them (customer info, order specification, or lettertext). Can I set react-pdf to use a pdf-file as starting point and then draw stuff on that? 

TIA.

**Comment by kishaningithub** [comment]
Linking the issue where the discussion on this happened https://github.com/diegomura/react-pdf/issues/628
